src/olympia/amo/tests/__init__.py

```diff
@@ -355,6 +356,30 @@ def logout_api(self):
         self.defaults.pop('HTTP_AUTHORIZATION', None)
 
 
+class JWTAPITestClient(JWTAuthKeyTester, APIClient):
```

Comment:
I added this so we can switch out `APITestClient` for the JWT version without having to rewrite tests.